### PR TITLE
[AMEND] Amend indexing example to use Scope.

### PR DESCRIPTION
**Pull request by NikRimington**
- [AMEND] Amended the indexing example to demonstrate the use of a Scope wrapper when accessing an Umbraco Context

Based on response from Shannon on this forum thread https://our.umbraco.com/forum/using-umbraco-and-getting-started/102676-triggering-index-rebuild-via-hangfire-causes-objectdisposedexception-in-nucache

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for amending this article @NikRimington ! 😄 

Looks great to be, but I'll some more expertise to help with a proper review.
Will make sure it gets done ASAP.

File: Reference/Searching/Examine/examine-events.md [hunk 1]
```diff
@@ -95,6 +104,29 @@ So at this point we have done something along the lines of:
...
+                return nodeBeingIndexed.Ancestor != null ?  
+                    string.Join(" ", nodeBeingIndexed.AncestorsOrSelf().Select(n => n.Name)) :
+                    string.Empty;
+            }
+        }
+    }
+}
+```
+
+This example is here to highlight the user of the Scope Provider. A Scope is required for many things in Umbraco, from controlling database transactions to how service level events are handled to how nucache manages it's snapshots (and more). Although Umbraco tries to ensure there is a Scope available, this is not always the case, especially when working in background threads / events. As such, it is important to wrap calls to Umbraco Services/Contexts in a Scope, as without this unusual errors can occur, including the disposal of objects still required. *For more information on this, please see the answer to this [forum question](https://our.umbraco.com/forum/using-umbraco-and-getting-started/102676-triggering-index-rebuild-via-hangfire-causes-objectdisposedexception-in-nucache)*
```
Comment (marcemarc) [inline Reference/Searching/Examine/examine-events.md:128]:
```suggestion
:::note
This example is here to highlight the user of the Scope Provider. A Scope is required for many things in Umbraco, from controlling database transactions to how service-level events are handled to how nucache manages it's snapshots (and more). Although Umbraco tries to ensure there is a Scope available, this is not always the case, especially when working in background threads/events. As such, it is important to wrap calls to Umbraco Services/Contexts in a Scope, as without this unusual errors can occur, including the disposal of objects still required. *For more information on this, please see the answer to this [forum question](https://our.umbraco.com/forum/using-umbraco-and-getting-started/102676-triggering-index-rebuild-via-hangfire-causes-objectdisposedexception-in-nucache)*
:::
```

Comment (marcemarc) [reply Reference/Searching/Examine/examine-events.md:128]: Just thought we should wrap the explanation in a 'note' - I also think we should add a note to the Services + Helpers page or have a scope page generally to link to, to flag the information. But in the context of this PR the sooner we add the note to the docs the sooner we help people!

**Comment by NikRimington** [comment]
Have updated with that change @marcemarc :-)

**Comment by marcemarc** [comment]
Thanks @NikRimington for taking the time to update it here! very useful info! It is Merged, should appear in the docs the next time @jmayntzhusen or @sofietoft presses the magic button!

**Comment by jmayntzhusen** [comment]
Magic button pushed 😉

**Pull request merged**